src/library/Uno.Themes.WinUI.Markup/Theme/ComboBox.cs

```diff
@@ -282,10 +258,10 @@ public static class Background
 
 				public static class EditableDropDown
 				{
-					public static class GlyphForeground
+					public static class Foreground
 					{
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxEditableDropDownGlyphForeground")]
-						public static ResourceValue<Brush> Default => new("ComboBoxEditableDropDownGlyphForeground", true);
+						public static ResourceValue<Brush> Glyph => new("ComboBoxEditableDropDownGlyphForeground", true);
 					}
 				}
```

Comment:
Remove this and let's just add a `EditableGlyphForeground` property to the `DropDown` class